src/Haipa.IdentityDb/Stores/ClientStore.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Haipa.IdentityDb.Extensions;
+using IdentityServer4.Models;
+using IdentityServer4.Stores;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Haipa.IdentityDb.Stores
+{
+    public class ClientStore : IClientStore
```

Comment:
Is that really necessary? Does the Identity Server not already implement an EFCore based store?

Reply:
the clientstore was modified with a custom one (and models)